packages/admin-ui-docs/src/components/CodeBlockWrapper/useCopyCodeBlock.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import type { MouseEventHandler, ReactNode } from 'react'
+
+import { Button, IconCopySimple } from '@vtex/admin-ui'
+
+interface ToggleCodeButtonProps {
+  children: ReactNode
+  onClick: MouseEventHandler<HTMLButtonElement>
+}
```

Comment:
I think you can replace this interface with this one, it's simpler than always creating an interface with native button props. Same for the toggle code preview button

```suggestion
import { Button, ButtonProps, IconCopySimple } from '@vtex/admin-ui'
```